awx/api/serializers.py

```diff
@@ -3199,6 +3197,13 @@ def to_representation(self, obj):
 
     def get_summary_fields(self, obj):
         summary_fields = super(JobSerializer, self).get_summary_fields(obj)
+        if obj.internal_limit:
+            summary_fields['internal_limit'] = {}
+            if obj.internal_limit.startswith('shard'):
```

Comment:
same here, etc. Would like to keep 'shard' out of the UX (including API endpoints).